numerics/hermite3_body.hpp

```diff
@@ -63,6 +64,21 @@ BoundedArray<Argument, 2> Hermite3<Argument, Value>::FindExtrema() const {
       arguments_.first, a1_, 2.0 * a2_, 3.0 * a3_);
 }
 
+template<typename Argument, typename Value>
+template<typename Samples, typename GetArgument, typename GetValue>
+typename Normed<Difference<Value>>::NormType
+Hermite3<Argument, Value>::LInfinityError(Samples const& samples,
+                                          GetArgument get_argument,
+                                          GetValue get_value) const {
```

Comment:
Pass the functors by reference, you are not moving them.

Reply:
Done.